### DISCUSSION TITLE
Capstone DA - Flask Dashboard UI (VBA tidak ada terdapat extensions Anaconda atau Mini Conda)

### DISCUSSION THREAD

**Discussion by vinsensiaellen** [teamalgoritma/community#755] [Q&A] [answered]
Tidak terdapat extensions Anaconda atau Mini Conda pada VBA saat mau menginstall sehingga tidak bisa dilakukan Run
![image](https://github.com/teamalgoritma/community/assets/143764255/757a5792-419d-43d6-af42-4dfa1378e63c)
![image](https://github.com/teamalgoritma/community/assets/143764255/57a841bc-bf72-4c06-9fdc-47806662611e)

Sehingga saat mau running, hasil yang muncul adalah sbg:
![image](https://github.com/teamalgoritma/community/assets/143764255/6746af0e-c141-41aa-ba7e-9d84590b32da)

**Comment by divakartika** [comment]
Terkait ekstensi Anaconda tidak perlu diinstall. Sudah solved di sesi Capstone DA ya, Bu. Semoga membantu, terima kasih :)

**Reply by lhpratama** [reply]
![ViscodeActivateErrorEnvironment](https://github.com/teamalgoritma/community/assets/141613272/a928fdf9-72dd-436a-974d-2f32a271a25d)

Mbak, saya mau nanya, saya kalau nyoba activate seperti biasa kayak gini juga error itu kenapa ya? Mohon bantuannya

**Reply by divakartika** [reply, marked as answer]
Dear @lhpratama 

Jika tidak dapat melakukan conda activate, kemungkinan karena saat ini Anda menggunakan terminal PowerShell. Dapat dipilih terlebih dahulu terminal yang akan digunakan, yaitu Command Prompt.
![image](https://github.com/teamalgoritma/community/assets/79572421/f75c52f2-c72a-4db2-a0c7-123108cf461a)

Untuk setting Command Prompt sebagai default terminal, dapat klik Select Default Profile dan pilih Command Prompt
![image](https://github.com/teamalgoritma/community/assets/79572421/23746035-1bcd-4a1a-b046-c5103aed63c1)
![image](https://github.com/teamalgoritma/community/assets/79572421/4a216490-5d21-4201-a341-3b580d773c85)

Jika sudah menggunakan Command Prompt maka dapat dituliskan `conda activate nama_environment` seperti biasa

![image](https://github.com/teamalgoritma/community/assets/79572421/8fd5bbb0-8441-49bb-9eb6-83d82f51c1c7)

Semoga membantu dan menjawab pertanyaan Anda :)
